docs/website/docs/user-guides/v3-migration-guide.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Migrate from v2 to v3
+sidebar_position: 2
+---
+
+### Migrate existing odo component from v2 to v3
+If you have created an odo component using odo v2, this section will help you move it to use odo v3.
+1. `cd` into the component directory, and delete the component from the Kubernetes cluster:
+    ```shell
+    odo delete
+   ```
+2. Download and install odo v3 by following the steps mentioned [here](../overview/installation.md).
+3. Use [`odo dev`](../command-reference/dev.md) to start developing your application using odo v3.
+4. You can also use `odo list` to see a list of components that are running on the cluster and which version of odo
+   was used to create them.
+
+If you face any problem, [open an issue on odo's repository](https://github.com/redhat-developer/odo/issues/new?assignees=&labels=&template=Bug.md).
```

Comment:
Should remove this? This is implied from our contributing / community section.

Reply:
Removed.